### DISCUSSION TITLE
Triggers Un-Remappable to M1 M2?

### DISCUSSION THREAD

**Discussion by xenobyteOG** [Valkirie/HandheldCompanion#959] [General]
Am I doing something wrong or are the triggers still not able to be remapped to M1 and M2? The app is great don't get me wrong but this is literally the only reason I downloaded it LOL. Are there any plans to make this work? Thank you in advance Thank you in advance.

**Comment by CasperH2O** [comment]
Hey @dpmalito thanks for dropping in. At the moment you cannot map an analogue signal (0 to 100% like the triggers) to a single press button HC. You can remap other buttons though.

**Reply by xenobyteOG** [reply]
Thanks for the response! I hope someday we can figure out how to do it well I mean you lol haha not me. 😀 It's funny because the PlayStation elite controller doesn't do it but the Xbox elite controller does so I usually use that on PC.
